gradient-color-picker/GradientColorProvider.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Drawing;
+using System.Linq;
+
+namespace Messerli.GradientColorPicker
+{
+    public class GradientColorProvider : IGradientColorProvider
+    {
+        private readonly IReadOnlyCollection<GradientColorByValue> _gradientColorList;
+
+        public GradientColorProvider(IImmutableList<GradientColorByValue> gradientColorList)
+        {
+            _gradientColorList = gradientColorList.OrderBy(item => item.Value).ToImmutableList();
+        }
+
+        public Color PickColor(int value)
+        {
+            if (!_gradientColorList.Any())
```

Comment:
Put this into a function

Reply:
Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/1/commits/070eb06dbe23669a7311f7a81e1ad4f94db22c2c

Reply:
Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/1/commits/070eb06dbe23669a7311f7a81e1ad4f94db22c2c